kube-client/src/api/portforward.rs

```diff
@@ -240,6 +257,9 @@ where
 {
     // Keep track if the channel has received the initialization frame.
     let mut initialized = vec![false; 2 * ports.len()];
+    let mut shutdown = vec![false; 2 * ports.len()];
```

Comment:
I think I would prefer to keep something like a `Vec<ChannelState>` where

```rust
struct ChannelState {
  initialized: bool,
  shutdown: bool,
}
```

rather than adding more and more vecs here.